python/ray/dashboard/client/src/App.tsx

```diff
@@ -80,6 +80,10 @@ export type GlobalContextType = {
    * running as detected by the grafana healthcheck endpoint.
    */
   grafanaHost: string | undefined;
+  /**
+   * The param 'orgId' used in grafana. Default is 1.
+   */
+  grafanaOrgId: string | undefined;
```

Comment:
should we just not allow "undefined"?

If we don't receive it from the API, we assume it's "1". 

Should orgId be a string or a number?